src/components/Wallet/Exchange/ExchangeShowMore/index.js

```diff
@@ -49,7 +49,7 @@ function ExchangeShowMore({
                   {' '}
                   {new BN(lpTokensBalance).mul(BN_HUNDRED).div(new BN(liquidity)).toString()}
                   % (
-                  {lpTokensBalance}
+                  {lpTokensBalance.toString()}
```

Comment:
Hmmm... lpTokensBalance is a BN here, right? I'd worry that `.toString()` may return hex for some values here. Maybe use formatAssets with decimals=0 instead?
Also, what values are seen here in practice? I'd worry that these are gigantic number, while at the same time it doesn't really make sense to put arbitrary decimals on it... Is it needed at all? Just keeping percent seems enough.

Reply:
yeah just keep percent then